controller/src/main/java/io/pravega/controller/server/rpc/grpc/impl/GRPCServerConfigImpl.java

```diff
@@ -73,6 +77,11 @@ public GRPCServerConfigImpl(final int port, final String publishedRPCHost, final
         this.authorizationEnabled = authorizationEnabled;
         this.userPasswordFile = userPasswordFile;
         this.tlsEnabled = tlsEnabled;
+        if (tlsProtocolVersion == null) {
+            this.tlsProtocolVersion = Config.TLS_PROTOCOL_VERSION;
```

Comment:
`Config.TLS_PROTOCOL_VERSION `already has a default value. This default value is passed while building the GRPCServiceConfig and hence this is never passed as null right? 
If yes, can you simply add a `Preconditions.checkNull(...)` for the tlsProtocolVersion?

Reply:
Fixed.Implemented this RESTServerConfigImpl with builder pattern.

Reply:
please remove redundant changes in test file due to this change.